lib/web_ui/lib/src/engine/initialization.dart

```diff
@@ -185,7 +186,7 @@ Future<void> initializeEngineServices({
         // milliseconds as a double value, with sub-millisecond information
         // hidden in the fraction. So we first multiply it by 1000 to uncover
         // microsecond precision, and only then convert to `int`.
-        final int highResTimeMicroseconds = (1000 * highResTime).toInt();
+        final int highResTimeMicroseconds = (1000 * highResTime.toDart).toInt();
```

Comment:
Does this throw a compile-time error if I forget to do `.toDart`? (I assume yes, but would like to confirm).

Reply:
It will on Dart2Wasm right away, and it will on JS backends after https://dart-review.googlesource.com/c/sdk/+/283742 lands, which will hopefully be soon.